packages/flyout/src/Flyout.js

```diff
@@ -59,10 +65,22 @@ class Flyout extends Component<Props> {
     onStateChange(this.state.isOpen);
   }
 
+  closeFlyout = () => {
+    const { onStateChange } = this.props;
```

Comment:
deze eventlistener blijft hangen als uw component unmount.
waarom wordt dit eigenlijk niet afgehandeld in de componentDidMount en ComponentWillUnmount?

Als de Flyout intieel open gerenderd wordt, dan reageert hij ook niet op een outsideClick?

Reply:
Sorry voor de vertraging op deze PR, project lag een paar dagen stil. Ik heb je feedback toegepast :)